release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -557,6 +570,19 @@ for more details.
 [id="ocp-4-5-bug-fixes"]
 == Bug fixes
 
+* Previously, it was difficult to see the list of Pods or resources associated with a Knative service in the *Topology* view. With this bug fix, when you select the Knative service, the sidebar displays a list of Pods along with a link to see the logs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1801752[*BZ#1801752*])
+* When you edited an existing query using the promQl editor in the *metrics* tab of the *Monitoring* view, the cursor moved to the end of the line. With this bug fix, the promQl editor works as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1806114[*BZ#1806114*])
```

Comment:
promQL (The "L" should be capitalized, I think)

Reply:
Apparently the P should be capitalized too

Reply:
Fixed